content/sse/index.textile

```diff
@@ -101,178 +98,6 @@ An important thing to note here is that the EventSource API tries to auto-reconn
-
-h5(#envelope-format). Envelope format
-
-Once a streaming response is established, every line (other than empty lines sent as keepalive packets) will be a simple JSON object of the following form:
-
-bc[json]. {
-  event: <string, the event type, such as message, presence, error or heartbeat>,
-  data: <message, presence or error object. Not present for heartbeats>,
-  id: <string, the ID to use to resume from this point, see connection state
-       recovery for details. Only present for message and presence events>
-}
-
-If @enveloped@ is true (the default), the @data@ will be a JSON-stringified object of a type determined by the event:
-
-- For a @message@ event it will be a "Message":/realtime/types#message object
-- For a @presence@ event it will be a "PresenceMessage":/realtime/types#presence-message object
-- For an @error@ event it will be an "ErrorInfo":/realtime/types#error-info object
-
-For a payload that is anything other than a string, as there is no client library to decode the payload of a @Message@ or @PresenceMessage@, you will have to decode it yourself. Objects and arrays will be json-encoded; binary payloads will be base64-encoded. The @encoding@ field of the @Message@ will specify what encoding has been used for the @data@ payload.
-
-If @enveloped@ is false, the @data@ will for a @message@ or @presence@ event be the @data@ payload from the @Message@ or @PresenceMessage@. Other events are unaffected.
-
-Non-string payloads will be encoded as before, but without enveloping you will not have the benefit of the @encoding@ field to tell you what encoding has been done.
-
-Note that failures on opening the connection (for example, invalid authentication details) may be sent as a non-streamed http response (with a response body of the form @{"error": <ErrorInfo>}@), not an @error@ event in a streamed response.
-
-h5. Code example
-
-```[curl]
-curl "https://rest.ably.io/event-stream?channel=example&v=1.2" \
-  --user "{{API_KEY}}"
-
-{
-  "id":"cbfKayrzgAXDWM:1556804156735-0",
-  "event":"message",
-  "data":{
-    "id":"oZs6XaGYx8:0:0",
-    "name":"message-name",
-    "timestamp":1556804156730,
-    "encoding":"json",
-    "channel":"example",
-    "data":"{\"foo\":1}"
-  }
-}
-⏎
-{
-  "event":"error",
-  "data":{
-    "message":"Token expired. (See https://help.ably.io/error/40142 for help.)",
-    "code":40142,
-    "statusCode":401,
-    "href":"https://help.ably.io/error/40142"
-  }
-}
-```
-
-```[nodejs]
-const request = require('request');
-const apiKey = '{{API_KEY}}';
-const url = 'https://realtime.ably.io/event-stream?channels=myChannel&v=1.2&key=' + apikey;
-
-request
-  .get(url)
-  .on('response', function(response) {
-    console.log(response.statusCode) // stream started, 200
-  })
-  .on('data', function(data) {
-    console.log('Envelope:', data.toString())
-  });
-```
+View the "Server-Sent Events object":/api/sse for the associated Client Library SDK API Reference.
```

Comment:
I think we should drop "Client Library SDK" here and also not refer to it as an object.

Reply:
Good call. Fixed.